apps/vaporgui/gui.pro

```diff
@@ -31,13 +31,14 @@ win32:QMAKE_CXXFLAGS_DEBUG += /EHsc
 win32:QMAKE_CXXFLAGS_RELEASE += /EHsc
 DEFINES += VOLUMIZER
 
+QT += widgets
 
 INCLUDEPATH += . \
 	../../include \
 	../../../apps/vaporgui/misc \
 	../../../apps/vaporgui/guis \
 	../../../apps/vaporgui/main \
-	../../../apps/vaporgui/render
+	../../../apps/vaporgui/render \
```

Comment:
What are these pointing to?

Reply:
This must be legacy code that has slipped through the cracks.  Removed.